server/services/store/sqlstore/migrations/000026_create_preferences_table.up.sql

```diff
@@ -0,0 +1,14 @@
+create table {{.prefix}}preferences
+(
+    userid   varchar(36) not null,
+    category varchar(32) not null,
+    name     varchar(32) not null,
+    value    text        null,
+    primary key (userid, category, name)
+    );
+
+create index idx_{{.prefix}}preferences_category
+    on {{.prefix}}preferences (category);
+
+create index idx_{{.prefix}}preferences_name
+    on {{.prefix}}preferences (name);
```

Comment:
We'd use them in personal server only as the table is not used in plugin mode. I created the indexes referrencing the `preferences` table from Mattermost server.  I agree that we don't need them as much as the Mattermost server does, but it might be useful for a personal server running for a small team, for example.

I don't see any harm in keeping them though I'm 1/5.